### PR TITLE
Leaderboard: API Consumption

### PR DESCRIPTION
**Pull request by zemola**
### **Leaderboard: API Consumption**

For this stage of the project, the following requirement was met.

- Reading the Leaderboard API documentation.
- Creating a new game by using the API.
- Implementing the "Refresh" button (receiving data from the API and parsing the JSON).
- Implementing the form "Submit" button (sending data to the API).
- Using async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/uoLjz1rr0ajNeJ1HqpM7/scores/';
+
+async function getScores() {
+  const response = await fetch(url);
+  const data = await response.json();
+  return data.result;
+}
+
+async function addScore(obj) {
```
Comment (TedLivist) [inline src/api.js]: - [ ] Let's use ES6 syntax for our functions as it is a requirement. Kindly change your functions here to arrow functions.

File: src/index.js [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
-import data from './data.js';
-import Input from './Input.js';
+import input from './Input.js';
+import { getScores, addScore } from './api.js';
 
-Input(data);
\ No newline at end of file
+const btn = document.querySelector('.submit-btn');
+
+async function mainRender() {
```
Comment (TedLivist) [inline src/index.js]: - [ ] Use ES6 syntax for your function.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @zemola,

Good job so far!
✔️ Your app UI looks good
✔️ Your app works as expected
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (zemola) [reply src/index.js]: Thank you,
Will change the function to arrow function as requested.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @zemola,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**